src/api/post/asyncGetPosts.js

```diff
@@ -0,0 +1,18 @@
+import fetchHandler from "..";
+import { BASE_URL, POST_LISTS } from "../../config/constants";
+
+const asyncGetPosts = async (
+  cursorId = POST_LISTS.CURSOR_ID,
+  { keywordId, includedKeyword = POST_LISTS.DEFAULT_INCLUDED_KEYWORD, limit = POST_LISTS.LIMIT }
```

Comment:
```suggestion
  cursorId = EMPTY_STRING_VALUE,
  { keywordId, includedKeyword = EMPTY_STRING_VALUE, limit = POST_LISTS.LIMIT }
```
기본값 지정을 위해 상수를 활용하는 방향이 좋은 것 같아요. 다만, 상수명을 변경해보는 건 어떨까요?
변수 각각의 이름을 따서 짓다보니 해당 값이 어떤 값인지 알지 못해, 상수 파일로 결국 가서 코드를 확인하게 되더라구요.

`EMPTY_STRING_VALUE` 상수명을 사용하면, `cursorId`, `includedKeyword`의 기본값으로 빈 string `""`을 넣고자 한 의도가 더 담길 수 있다고 생각해요. 상수 파일에서 직접 코드를 확인하는 수고도 줄일 수 있고요. 어떻게 생각하시나요? 🤔

Reply:
골똘히 생각하게 만드는 리뷰... 감사합니다 🙆‍♂️ 생각해본 결과 장단점이 있을 것 같아요. 

- `EMPTY_STRING_VALUE`로 바꿔주면 가독성이 좋아져서 변수명만 보고 유추가 되는 반면, 추후에 `cursorId`와 `includedKeyword`의 기본값이 변경되었을 때, 새로운 매직 넘버에 대한 변수명을 추가로 만들어줘야 한다는 점이 있을 수 있겠습니다. 저 같은 경우는 빈 문자열은 그냥 ""로 표시를 하는 편입니다.
- 반면, 제가 작성한 버전에서는 상수파일만 가면 그곳에 객체로 관심사 분리가 되어 있어서 해당 POST_LISTS 관련 변수들을 빠르게 찾아볼 수 있고, 추후 변경이 되었을 때, 수정이 용이하다는 장점이 있는 반면, 변수명을 보고 한 뎁스 들어가서 찾아봐야한다는 단점이 있겠네요.

@only-pine 어떻게 생각하시나요?

Reply:
규경님 말씀이 이해가 되었어요. 추후 수정, 확장성을 위해 현재 버전을 유지해도 좋겠습니다! 
그리고 `EMPTY_STRING_VALUE`로 한다면 따로 상수로 두지 않고 ""로 표시하면 되겠네요~!

Reply:
저도 사실 해당 코드를 봤을 때 default값을 설정하신게 맞는건지 잠깐 생각했었어요. 아무래도 prettier 때문에 줄바꿈때문에 그렇게 느껴졌을 뿐만 아니라 상수명이 디폴트 값을 의미하고 있지 않아서 더 그랬던 것 같아요..!
진태님 의견대로 상수명을 바꾸되, 규경님 의견처럼 빈문자열의 경우엔 `""` 로 표시하고 limit의 경우에만 `DEFAULT_POSTS_LIMIT`, `DEFAULT_LIMIT` 와 같이 default값이란 것을 알려주는 상수명으로 변경하면 더 직관적이지 않을까 생각합니다!

@SsongQ-92 @jin-ttao

Reply:
@only-pine @jin-ttao 두 분 의견 반영하여 아래와 같이 변경하도록 하겠습니다.

```js
export const POST_LISTS = Object.freeze({
  DEFAULT_INCLUDED_KEYWORD: "",
  DEFAULT_LIMIT: 10,
  DEFAULT_CURSOR_ID: "",
});
```

```js
const asyncGetPosts = async (
  cursorId = POST_LISTS.DEFAULT_CURSOR_ID,
  { keywordId, includedKeyword = POST_LISTS.DEFAULT_INCLUDED_KEYWORD, limit = POST_LISTS.DEFAULT_LIMIT }
) => {
  const fetchInfo = {
    url: `${BASE_URL}/posts/${keywordId}`,
    params: `?includedKeyword=${includedKeyword}&limit=${limit}&cursorId=${cursorId}`,
  };
```